src/frontend/src/eth/stores/eth-transactions.store.ts

```diff
@@ -41,4 +41,4 @@ const initTransactionsStore = (): TransactionsStore => {
 	};
 };
 
-export const transactionsStore = initTransactionsStore();
+export const ethTransactionsStore = initTransactionsStore();
```

Comment:
You will rename the rest of the module, such as its interfaces, in a separate PR?

Reply:
correct, I am doing it slowly, just to avoid any conflicts with other PRs